libraries/aws/defender/include/aws_iot_defender.h

```diff
@@ -65,13 +65,19 @@
  * @brief Bit flags or metrics used by @ref defender_function_setmetrics function.
```

Comment:
why is there any change in defender?

Reply:
May have been an artifact from an older `v4_beta` sync. Good catch. Overwrote with the latest version from `v4_beta`